xonsh/completers/path.py

```diff
@@ -280,7 +284,7 @@ def complete_path(prefix, line, start, end, ctx, cdpath=True, filtfunc=None):
     if p is not None:
         lprefix = len(p[0])
         prefix = p[1]
-        path_str_start = p[2]
+        path_str_start = p[2].lstrip("p")
```

Comment:
I think this might do the wrong thing for `fp""` strings

Reply:
Nothing changes for the case when the quote is `pf"` or `fp"`, as `_path_from_partial_string` returns `None` for these cases. More than happy to add a check before stripping though.